README.md

```diff
@@ -1 +1,70 @@
-# book-management
\ No newline at end of file
+# Book-management
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn start`
+
+Runs the app in the development mode.<br />
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.<br />
+You will also see any lint errors in the console.
+
+### `yarn test`
+
```

Comment:
please clean up this file. let your `readme` be project specific and not the default provided by react. Add things like the setup info, contributor(s) etc.